_gsocproposals/2019/proposal_HTCondorTransferTests.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Evaluation of scalability of protocols to serve data for jobs for LIGO/Collaboration
+layout: gsoc_proposal
+project: HTCondor
+year: 2019
+organization: UCSD
+---
+
+## Description
+
+The LIGO experiment is adopting the use of GlideinWMS, a distributed resource manager, for its computing needs. Currently, passing input data to the parallel jobs is accomplished via either HTCondor-based file transfer or downloading from a set of Xrootd-powered distributed caches. Past work has been done on testing the scalability of these transfer methods, but not in the context of the needs of the LIGO experiment. We will attempt to test at what concurrency levels different components of the GlideinWMS infrastructure break while keeping conditions aligned to the needs of the LIGO experiment. 
+
+
+## Task ideas
```

Comment:
Could you please fix the external links as `[Label](URL)`?

Reply:
@aflp91 sorry for that I update them.